pkg/roachprod/prometheus/prometheus.go

```diff
@@ -168,24 +176,11 @@ rm -rf node_exporter && mkdir -p node_exporter && curl -fsSL \
 			`cd node_exporter &&
 sudo systemd-run --unit node_exporter --same-dir ./node_exporter`,
 		); err != nil {
-			return nil, err
-		}
-
-		// Add a scrape config for each node exporter
-		for i, node := range cfg.NodeExporter {
-			s := strconv.Itoa(int(node))
-			cfg.ScrapeConfigs = append(cfg.ScrapeConfigs, ScrapeConfig{
-				JobName:     "node_exporter-" + s,
-				MetricsPath: "/metrics",
-				Labels:      map[string]string{"node": s},
-				ScrapeNodes: []ScrapeNode{{
-					Nodes: install.Nodes{node},
-					IPs:   []string{cfg.NodeExporterIPs[i]},
-					Port:  9100}},
-			})
+			// TODO: systemd-run cannot run on mac, so this likely errors when people
```

Comment:
Leave your name on this TODO? Also it's not just systemd, the binaries we're download above are platform specific.

Reply:
Done. I didn't leave my name because I don't want to work on that :))))))))